### PR TITLE
add test to check successful container restart

### PR DESCRIPTION
**Pull request by philipkozeny**
provides a test to check if the API still works after restarting the container. As discussed in https://github.com/mediagis/nominatim-docker/issues/214

### CODE REVIEW

**Comment by leonardehrenfried** [comment]
I need to experiment a bit but perhaps it's because of this line: https://github.com/mediagis/nominatim-docker/blob/master/3.7/init.sh#L43

The error message suggests something else though.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -68,10 +68,22 @@ jobs:
...
             -v /tmp/nominatim-postgres:/var/lib/postgresql/12/main \
-            -p 8082:8080 \
+            -p 8003:8080 \
             nominatim &
           sleep 35
-          ./assert-non-empty-json "http://localhost:8082/search.php?q=avenue%20pasteur"
+          ./assert-non-empty-json "http://localhost:8003/search.php?q=avenue%20pasteur"
+
+      - name: Check container restart
+        working-directory: .github/workflows
```
Comment (leonardehrenfried) [inline .github/workflows/ci.yml:77]: You're using the same volume with two running containers. Is this a supported usecase? Wouldn't it be wiser to stop the previous container?

Comment (philipkozeny) [reply .github/workflows/ci.yml:77]: True, thanks for the update in your PR.

**Comment by philipkozeny** [comment]
Closed in favour of #216 where those changes are included.

**Pull request closed without merging**